src/CloudModelGenerator/Program.cs

```diff
@@ -16,6 +16,7 @@ static int Main(string[] args)
             var projectIdOption = app.Option("-p|--projectid", "Kentico Cloud Project ID.", CommandOptionType.SingleValue);
             var namespaceOption = app.Option("-n|--namespace", "Namespace name of the generated classes.", CommandOptionType.SingleValue);
             var outputDirOption = app.Option("-o|--outputdir", "Output directory for the generated files.", CommandOptionType.SingleValue);
+            var transformFilenamesOption = app.Option("-f|--transformfilenames", "Output directory for the generated files.", CommandOptionType.SingleValue);
```

Comment:
The help message wasn't updated. Suggest something like "Optionally add a suffix to generated filenames (e.g., News.cs becomes News.Generated.cs)"